lib/pages/settings/view/request_card.dart

```diff
@@ -0,0 +1,207 @@
+                                      },
+                                    )
+                                  ],
+                                )
+                              else if (request.processed == true)
+                                Row(
+                                  mainAxisAlignment: MainAxisAlignment.end,
+                                  children: [
+                                    AppButton(
+                                      text: S.current.stringRevert,
+                                      color: Theme.of(context).accentColor,
+                                      width: 100,
+                                      onTap: () async {
+                                        await adminProvider
+                                            .revertAcceptedRequest(
+                                                request.formId, request.userId);
+                                      },
+                                    )
+                                  ],
+                                )
+                              else if (request.processed == null)
+                                Row(
+                                  mainAxisAlignment: MainAxisAlignment.end,
+                                  children: [
+                                    AppButton(
+                                      text: S.current.stringRevert,
+                                      color: Theme.of(context).accentColor,
+                                      width: 100,
+                                      onTap: () async {
+                                        await adminProvider.revertDeniedRequest(
+                                            request.formId, request.userId);
+                                      },
+                                    )
+                                  ],
+                                )
+                            ],
+                          ),
+                        ),
+                      ),
+                    );
+                  } else {
+                    return const SizedBox.shrink();
+                  }
+                });
+          } else {
+            return const SizedBox.shrink();
+          }
+        });
+  }
+
+  BoxDecoration boxDecorationGreen() {
+    return BoxDecoration(
+      border: Border.all(color: Colors.green, width: 2),
+      borderRadius: const BorderRadius.all(
+          Radius.circular(6) //                 <--- border radius here
+          ),
+    );
+  }
+
+  BoxDecoration boxDecorationRed() {
+    return BoxDecoration(
+      border: Border.all(color: Colors.red, width: 2),
+      borderRadius: const BorderRadius.all(
+          Radius.circular(6) //                 <--- border radius here
+          ),
+    );
+  }
+
+  @override
+  bool get wantKeepAlive => true;
+
+  @override
+  dynamic noSuchMethod(Invocation invocation) => super.noSuchMethod(invocation);
```

Comment:
Why is this necessary?

Reply:
Please answer comments before marking them as resolved, I'm not a mind reader :)

Reply:
Requested by AutomaticKeepAliveClientMixin